gossip/c_block_callbacks.go

```diff
@@ -380,9 +384,16 @@ func consensusCallbackBeginBlockFn(
 
 					now := time.Now()
 					blockAge := now.Sub(block.Time.Time())
-					log.Info("New block", "index", blockCtx.Idx, "id", block.Hash(), "gas_used",
-						evmBlock.GasUsed, "txs", fmt.Sprintf("%d/%d", len(evmBlock.Transactions), len(skippedTxs)),
-						"age", utils.PrettyDuration(blockAge), "t", utils.PrettyDuration(now.Sub(start)))
+					log.Info("New block",
+						"index", blockCtx.Idx,
+						"id", block.Hash(),
+						"gas_used", evmBlock.GasUsed,
+						"gas_rate", float64(evmBlock.GasUsed)/blockDuration.Seconds(),
+						"base_fee", evmBlock.BaseFee.String(),
+						"txs", fmt.Sprintf("%d/%d", len(evmBlock.Transactions), len(skippedTxs)),
+						"age", utils.PrettyDuration(blockAge),
+						"t", utils.PrettyDuration(now.Sub(start)),
```

Comment:
This start value is misleading, we may want to remove it.
It would be more interesting to have the blockDuratin and the blockctx.time here. 
After all, the timestamp is introduced by the logging itself.

Reply:
This log is only partially targeted for human consumption. It is, unfortunately, also used by Norma to collect monitoring data from the client. The fields, as they have been used before and the ones added, are needed in Norma to plot base-fee prices and throughput rates over time.